Offre.Test/User/UserLogicTests.cs

```diff
@@ -12,77 +15,194 @@ namespace Offre.Test.User
     [TestClass]
     public class UserLogicTests
     {
-        private readonly Mock<DbSet<UserModel>> _userSetMock = new Mock<DbSet<UserModel>>();
         private readonly Mock<IOffreContext> _offreContextMock = new Mock<IOffreContext>();
 
         private UserLogic GetTestSubject()
         {
             return new UserLogic(_offreContextMock.Object);
         }
 
-        //[TestMethod]
-        //public void GetAllUsers()
-        //{
-        //    var userLogic = GetTestSubject();
+        [TestMethod]
+        public void GetAllUsers_GetAllUsersFromTable_ReturnsValidUserCount()
+        {
+            var userLogic = GetTestSubject();
 
+            var userList = new List<UserModel>
+            {
+                new UserModel
+                {
+                    Id = 12
```

Comment:
If there is no reason for using a random, unknown number for id, begin from the 0 and increment it by one.

Reply:
Done